Integrations/integration-opswat-metadefender.yml

```diff
@@ -39,42 +39,82 @@ configuration:
   defaultvalue: "66"
   type: 0
   required: true
+- display: It is cloud
```

Comment:
I would change it to something which looks a bit more professional, like `Cloud based`

Reply:
Changed